src/bumper.h

```diff
@@ -0,0 +1,34 @@
+#include "lrf.h"
+#include <std_msgs/Bool.h>
+
+
+class Bumper : public LRF
```

Comment:
Bumper is geen subclasse van LRF. Je kunt wel de bumper zo maken dat ie een LRF als element heeft.